docs/src/workflow examples/Composition/composition.jl

```diff
@@ -0,0 +1,74 @@
+# # Model Composition with MLJFlux
+
+# In this workflow example, we see how MLJFlux enables composing MLJ models with MLJFlux models. We will assume a 
+# class imbalance setting and wrap an oversampler with a deep learning model from MLJFlux.
+
+using Pkg     #src
+Pkg.activate(@__DIR__);     #src
+Pkg.instantiate();     #src
+
+# **Julia version** is assumed to be 1.10.*
+
+# ### Basic Imports
+
+using MLJ               # Has MLJFlux models
+using Flux              # For more flexibility
+import RDatasets        # Dataset source
+import Random           # To create imbalance
+import Imbalance        # To solve the imbalance
+
+# ### Loading and Splitting the Data
+
+iris = RDatasets.dataset("datasets", "iris");
+y, X = unpack(iris, ==(:Species), colname -> true, rng=123);
+X = Float32.(X);      # To be compatible with type of network network parameters
+
+# To simulate an imbalanced dataset, we will take a random sample:
+Random.seed!(803429)
+subset_indices = rand(1:size(X, 1), 100)
+X, y = X[subset_indices, :], y[subset_indices]
+Imbalance.checkbalance(y)        
+
+
+
+# ### Instantiating the model
+
+# Let's load `BorderlineSMOTE1` to oversample the data and `Standardizer` to standardize it.
+BorderlineSMOTE1 = @load BorderlineSMOTE1 pkg=Imbalance verbosity=0
+NeuralNetworkClassifier = @load NeuralNetworkClassifier pkg=MLJFlux
+Standardizer = @load Standardizer pkg=MLJModels
```

Comment:
This is not incorrect, but `Standardizer` is always available without needing to explicitly load it. See all built-in models by doing `local models()` at startup.

Reply:
OK. I will add a comment next to it to clarify it's not necessary.

Reply:
Or I can remove the line and justify with a comment